linker/Mono.Linker/Driver.cs

```diff
@@ -225,10 +225,12 @@ static IStep ResolveStep (string type)
 		static string [] ReadLines (string file)
 		{
 			ArrayList lines = new ArrayList ();
-			using (StreamReader reader = new StreamReader (file)) {
-				string line;
-				while ((line = reader.ReadLine ()) != null)
-					lines.Add (line);
+			using (FileStream fileStream = new FileStream (file, FileMode.Open)) {
+				using (StreamReader reader = new StreamReader (fileStream)) {
+					string line;
+					while ((line = reader.ReadLine ()) != null)
+						lines.Add (line);
+				}
 			}
```

Comment:
StreamReader(string) seems to be available in coreclr: https://github.com/dotnet/coreclr/blob/c35e8dbc37e5380f46553510d0368aad04a677d2/src/mscorlib/src/System/IO/StreamReader.cs#L160